src/com/facebook/buck/go/GoBuckConfig.java

```diff
@@ -109,7 +110,11 @@ Path getDefaultPackageName(BuildTarget target) {
       public Tool get() {
         Optional<Path> toolPath = delegate.getPath("go", configName);
         if (!toolPath.isPresent()) {
-          toolPath = Optional.of(goToolDirSupplier.get().resolve(toolName));
+          String exeName = toolName;
+          if (delegate.getPlatform() == Platform.WINDOWS) {
+            exeName = toolName + ".exe";
+          }
```

Comment:
Hmm, if we used an `ExectuableFinder`, it would do this for us (and also handle the case where someone may have wrapped it in a `.bat` or some other script first).